.gitlab/common/skip_ci_check.yml

```diff
@@ -0,0 +1,22 @@
+# Disallow [skip ci] in commit messages
+
+skip-ci-check:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  needs: []
+  stage: setup
+  tags: ["arch:amd64"]
+  rules:
+    - !reference [.except_mergequeue]
```

Comment:
Isn’t this the opposite of what is in the PR’s description:
> ### What does this PR do?
> Adds a skip ci check to the merge queue

With these rules, this check will run everywhere except on the merge queue.